src/frontend/package.json

```diff
@@ -7,7 +7,9 @@
     "build": "webpack",
     "build-dev": "yarn copy-iframe-resizer; yarn build",
     "copy-iframe-resizer": "cp node_modules/iframe-resizer/js/iframeResizer.min.js ../backend/marsha/static/js",
-    "lint": "tslint -c tslint.json '**/*.ts?(x)' -e 'node_modules/**/*'",
+    "lint": "tslint --fix -c tslint.json '**/*.ts?(x)' -e 'node_modules/**/*'",
```

Comment:
Does this exit with `1` when it does modifications?

The CI runs `yarn lint` to ensure `tslint` passes so if this returns `0` after fixing the code it means `tslint` will always pass on the ci

Reply:
hmmm if this is run by the CI we must not modify it. I'll rollback and change the strategy to `warn` everything

Reply:
Sorry, maybe I should have mentioned this earlier.

If you prefer write mode maybe we can make a `lint-fix` command to run tslint with fixes and bundle it in `lint-all`?